api/.env.testing

```diff
@@ -67,6 +67,7 @@ INTERNAL_NOTIFICATION_BACKEND=pcapi.notifications.internal.backends.slack.SlackB
 JWT_ADAGE_PUBLIC_KEY_FILENAME=public_key.testing
 METABASE_DASHBOARD_ID=438
 NATIVE_APP_MINIMAL_CLIENT_VERSION=1.166.3
+NOTIFICATION_PERIOD_BEFORE_DELETING_SUSPENDED_ACCOUNT=61  # The day of suspension (because DELETE_SUSPENDED_ACCOUNTS_SINCE = 61)
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏
```suggestion
NOTIFY_X_DAYS_BEFORE_DELETION=61 # The day of suspension (because DELETE_SUSPENDED_ACCOUNTS_SINCE = 61)